CA_DataUploaderLib/Redundancy.cs

```diff
@@ -0,0 +1,233 @@
+﻿using CA.LoopControlPluginBase;
+using System.Collections.Generic;
+using System;
+using CA_DataUploaderLib.IOconf;
+using System.Linq;
+using CA_DataUploaderLib.Extensions;
+
+namespace CA_DataUploaderLib
+{
+    public class Redundancy
+    {
+        public Redundancy(CommandHandler cmd)
+        {
+            var configs = IOconfRedundant.ToDecisionConfigs(IOconfFile.GetEntries<IOconfRedundant>());
+            cmd.AddDecisions(configs.Select(r => new Decision(r)).ToList());
+            SwitchBoardController.Initialize(cmd);
+        }
+
+        public static void RegisterSystemExtensions()
+        {
+            IOconfFileLoader.AddLoader("RedundantSensors", (row, lineNumber) => new IOconfRedundantSensors(row, lineNumber));
```

Comment:
please put into string properties in the relevant classes that can then be referenced :)